taichi/backends/opengl/codegen_opengl.cpp

```diff
@@ -258,8 +258,8 @@ class KernelGen : public IRVisitor {
   void visit(GetChStmt *stmt) override {
     emit("int {} = {} + {}; // {}", stmt->short_name(),
          stmt->input_ptr->short_name(),
-         struct_compiled_
-             ->class_get_map[stmt->input_snode->node_type_name][stmt->chid],
+         struct_compiled_->snode_map[stmt->input_snode->node_type_name]
```

Comment:
FYI: If you do not want to accidentally insert a default value into the map when the key is missing, consider using [`find`](http://www.cplusplus.com/reference/unordered_map/unordered_map/find/) over `operator[]`.

Reply:
I guess [`at`](www.cplusplus.com/reference/map/map/at) should also work?

Reply:
Yes (sorry i slipped through this PR..)